ciao-cli/main.go

```diff
@@ -188,11 +193,11 @@ func sendHTTPRequestToken(method string, url string, values []queryValue, token
 	infof("Got HTTP response (status %s)\n", resp.Status)
 
 	if resp.StatusCode >= http.StatusBadRequest {
-		respBody, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			errorf("Could not read the HTTP response %s\n", err)
+		respBody, errBody := ioutil.ReadAll(resp.Body)
```

Comment:
to fix linter error,

err is already defined in line 147
https://github.com/albertomurillo/ciao/blob/master/ciao-cli/main.go#L147

but maybe this change should go in a separate commit